test/molecule.sh

```diff
@@ -14,4 +20,6 @@ fi
 
 cd molecule-role
 
+echo =====MOLECULE_RUN_ID=${CI_JOB_ID}======AGENT_GITLAB_BRANCH=${CI_COMMIT_REF_NAME}===== $AWS_ACCESS_KEY_ID ==
```

Comment:
why do we need to print the `AWS_ACCESS_KEY_ID` ?
for security I would rather avoid if not essential, you can always check the settings in gitlab